src/utils/push_array.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Does `array.push` for all `items`.
+ * Needed because `array.push(...items)` throws
+ * "Maximum call stack size exceeded" when items
+ * is too big of an array
+ * @param {any[]} array 
+ * @param {any[]} items 
+ */
+ export function push_array(array, items) {
+    for (let i = 0; i < items.length; i++) {
+      array.push(items[i]);
+    }
+  };
```

Comment:
it looks like there's whitespace before the `}` which shouldn't be necessary. same above before the `export`

Reply:
I'm no longer able to format code since using Prettier 😄 Fixed